app-containers/lxd/lxd-6.1.ebuild

```diff
@@ -97,7 +97,7 @@ src_prepare() {
 
 	# Fix hardcoded ovmf file path, see bug 763180
 	sed -i \
-		-e "s:/usr/share/OVMF:/usr/share/edk2-ovmf:g" \
+		-e "s:/usr/share/OVMF:/usr/share/edk2/OvmfX64:g" \
```

Comment:
For all reverse dependencies where we change hardcoded paths in the build system: We should probably do a revision bump when changing the dependency to the new firmware package.

Reply:
As discussed, this is fine because there's a compatibility symlink. I'm not planning to drop that any time soon.

Reply:
I should add that I did look at the files involved and also found references to the arm64 firmware, so we should fix that up later too.